addons/xbmc.python/addon.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="xbmc.python" version="2.24.0" provider-name="Team-Kodi">
-  <backwards-compatibility abi="2.1.0"/>
+<addon id="xbmc.python" version="2.25.0" provider-name="Team-Kodi">
+  <backwards-compatibility abi="2.25.0"/>
```

Comment:
I am telling you that (most of) python people are not complete idiots. and that you should not bump python abi for just one addon that uses crap (you said that) api, exposed for years.

I am telling you nothing more, nothing less. and I am not a developer too. but I am not complete idiot, as well. 

so, do whatever you think makes sense ;)